x/ccv/provider/keeper/double_vote_test.go

```diff
@@ -209,7 +209,7 @@ func TestVerifyDoubleVotingEvidence(t *testing.T) {
 				),
 			},
 			chainID,
-			tmprotocrypto.PublicKey{},
```

Comment:
maybe still a valid negtive test with empty content. 
might be good to have both nil and {} ad pk for 'invalid pk tests'?